pkg/operator/operator.go

```diff
@@ -171,8 +191,8 @@ func (optr *Operator) eventHandler() cache.ResourceEventHandler {
 	}
 }
 
-// on deployments we only reconcile on update/delete events if its owned by mao
-func (optr *Operator) eventHandlerDeployments() cache.ResourceEventHandler {
+// we only reconcile on update/delete events if its owned by mao
+func (optr *Operator) eventHandlerOwned() cache.ResourceEventHandler {
```

Comment:
this should go into its own commit?
Is there a way to set a singleton informer via library rather than with the custom event handler?

Reply:
It's just a change for the method name, without changes in functionality.